lib/actions/find.js

```diff
@@ -21,47 +19,29 @@ const Boom = require('boom');
 
 module.exports = function findRecords(route, options) {
 
-    return (request, reply) => {
+    return async (request) => {
 
         const actionUtil  = require('../actionUtil')(request, options);
 
         // Look up the model
         const Model = actionUtil.parseModel();
-
         // Lookup for records that match the specified criteria.  Are we just counting?
         if (options._private.count) {
 
-            Model.query().count().asCallback((modelErr, modelCount) => {
-
-                if (modelErr) {
-                    return reply(Boom.wrap(modelErr));
-                }
-                //this feels fragile, but I don't see another way to get at the data
-                return reply(modelCount[0]['count(*)']);
-            });
-
+            const modelCount = await Model.query().count();
+            //this feels fragile, but I don't see another way to get at the data
+            return modelCount[0]['count(*)'];
```

Comment:
I think this varies between database adapters– give it a whirl on postgres and i think this comes back as `count` rather than `count(*)`.  Here's how they handle it in objection tests https://github.com/Vincit/objection.js/blob/dd53965a2ce0e5f195a1a96e46bed44c9ad2c478/tests/integration/find.js#L171-L177